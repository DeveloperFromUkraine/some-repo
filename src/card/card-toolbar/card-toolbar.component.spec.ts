import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardToolbarComponent } from "./index";
import { ComponentTest } from "../../../test/test-bed/component";


describe('Card Toolbar', () => {
    let fixture: ComponentFixture<CardToolbarComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardToolbarComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardToolbarComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});