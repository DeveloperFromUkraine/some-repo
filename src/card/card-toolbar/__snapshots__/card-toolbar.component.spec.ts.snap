// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardToolbarComponent should match snapshot 1`] = `
<ign-card-toolbar>
  <div
    class="right-actions"
  >
    
  </div>
</ign-card-toolbar>
`;
